duckduckgo/macos-browser
# Search with DuckDuckGo in PDF Context Menu

Task/Issue URL: https://app.asana.com/0/1199237043630360/1201320151390574/f
Tech Design URL:
CC: @tomasstrba  @samsymons 

**Description**:
Fixes "Search With %@" in PDF Plugin opening text search in Safari 

**Steps to test this PR**:
1. Open a PDF (https://www.apple.com/ca/mac/docs/Apple_T2_Security_Chip_Overview.pdf)
2. Right-click a text, click "Search With DuckDuckGo" - should open new tab with SERP
3. Open Safari Preferences, change Default Search Engine to something else, validate the PDF menu still displays "Search With DuckDuckGo"
4. Validate other PDF Context Menu items are working as intended, including when clicking an empty space

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
